docs/developer/taxes.mdx

```diff
@@ -0,0 +1,356 @@
+---
+title: Taxes
+---
+
+Saleor offers two ways of calculating taxes: Flat Rates and Dynamic Taxes.
+
+Flat Rates allow you to configure static tax rates per country and channel, and assign them to products via tax classes. Dynamic Taxes, on the other hand, enable you to delegate tax calculation to external apps, including 3rd-party tax providers like Avalara, TaxJar, or entirely custom tax apps.
+
+You can configure the tax calculation method per channel and override the configuration for different countries.
+
+## Tax Configuration
+
+The tax configuration object defines various configuration options for a channel. This includes the tax calculation method, whether to charge taxes in this channel, or whether prices are entered including tax. The tax configuration object is created automatically for each sales channel. In the API, it is represented by the `TaxConfiguration` object (see the [API reference](api-reference/objects/tax-configuration.mdx)).
+
+The tax configuration object has several properties, including:
+
+- `chargeTaxes` - Determines whether taxes are charged in the channel. If enabled, taxes are calculated and customers pay the gross prices. If disabled, taxes are still calculated and available for display, but customers pay the net prices.
+- `taxCalculationStrategy` - The strategy to use for tax calculation in the channel. There are two strategies available:
+  - flat rates - the default strategy
+  - tax apps (dynamic taxes)
+- `displayGrossPrices` - Determines whether prices displayed in a storefront should include taxes. Note that this setting doesn’t affect internal calculations in Saleor, and it’s only meant for the storefront to decide if it should display `TaxedMoney.net` or `TaxedMoney.gross` price (see [TaxedMoney API reference](api-reference/objects/taxed-money.mdx)). For storefront queries, this property can be fetched at two levels:
+  - product level in the [ProductPricingInfo.displayGrossPrices](api-reference/objects/product-pricing-info.mdx) field - use it when rendering product listings or product details pages
+  - checkout level in the [Checkout.displayGrossPrices](api-reference/objects/checkout.mdx) field - use it when rendering the cart or checkout pages.
+- `pricesEnteredWithTax` - Determine whether product prices are entered with tax included or not. If they are, assume that entered prices include tax (typical for B2C in EU). Otherwise, assume that prices exclude tax (typical for US and B2B).
```

Comment:
```suggestion
- `pricesEnteredWithTax` - Determine whether product prices are entered with tax included.
```

Isn't this kind of saying the same thing twice? AFAIK, the first sentence suffices.